### ISSUE TITLE
Issue with mask numbers for decimal places

### ISSUE THREAD

**Issue by ajgaunt** [miranj/craft-cryptographer#6]
It appears the mask numbers function only works for whole numbers, not numbers with decimal places. Can use the encrypt function but thought I'd just see if this was an item that could be updated.

**Comment by rungta** [comment]
That is correct @ajgaunt. The `maskNumbers` Twig filter is simply a wrapper around [HashIds which says][1]:

> The library currently supports only positive numbers and zero.

You could try putting in a feature request with them, perhaps? I’ll update the documentation to mention this nuance on the plugin's page.

[1]:https://hashids.org/php/#what-not-to-do
